### ISSUE TITLE
[Bug] (1.0.8) - Selecting Image gif Node Then Cancelling Crashes

### ISSUE THREAD

**Issue by Synthoid** [Ttanasart-pt/Pixel-Composer#21]
Opening the new node window and selecting `Input/Image gif` then cancelling while selecting a path causes Pixel Composer to throw this error and crash:

```
___________________________________________
############################################################################################
ERROR in
action number 1
of Draw Event
for object o_dialog_add_node:

argument is not a method, unable to call############################################################################################
gml_Script_build_NodeObject_gml_GlobalScript_node_registry (line 18)
gml_Script_buildNode_gml_Object_o_dialog_add_node_Create_0 (line 42)
gml_Script_anon_gml_Object_o_dialog_add_node_Create_0_3944838_gml_Object_o_dialog_add_node_Create_0 (line 177)
gml_Script_anon_scrollPane_gml_GlobalScript_scrollPane_768613_scrollPane_gml_GlobalScript_scrollPane (line 45)
gml_Object_o_dialog_add_node_Draw_64 (line 23)
```

![image](https://user-images.githubusercontent.com/6645950/210464076-24e02ee1-9aa6-4b61-a0bc-34c6c51e29b4.png)

**Comment by Ttanasart-pt** [comment]
Thanks for reporting! This one should be an easy fix.

**Comment by Ttanasart-pt** [comment]
Fixed in 1.0.9.
